### ISSUE TITLE
Material lettera icon 

### ISSUE THREAD

**Issue by Angelk90** [prscX/react-native-chip-view#5]
@prscX : Hi, looking at the module I saw that it has the possibility to have material letter icon, for android I found this interesting module: https://github.com/IvBaranov/MaterialLetterIcon/blob/master/README.md 
it would be interesting to be able to implement it as a feature .

**Comment by prscX** [comment]
Thanks a lot @Angelk90 for providing the library reference.

Today I have pushed release with new set of icons. I feel it will provide a neat look on the UI. Please let me know your view on the same.

Thanks
</ Pranav >

**Comment by Angelk90** [comment]
@prscX :  Yes, just because I saw the update you made, I thought of you. Personally I really love this style used by Google in Android, Gmail and Microsoft in Outlook.

**Comment by prscX** [comment]
Thanks @Angelk90. Completely agree with you, coming days I will integrate these icons family as well.

Thanks
</ Pranav >
